package.json

```diff
@@ -21,32 +21,30 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "2.4.3",
-    "@docusaurus/plugin-client-redirects": "2.4.3",
-    "@docusaurus/plugin-google-analytics": "2.4.3",
-    "@docusaurus/preset-classic": "2.4.3",
-    "@docusaurus/theme-mermaid": "2.4.3",
+    "@docusaurus/core": "3.0.1",
+    "@docusaurus/plugin-client-redirects": "3.0.1",
+    "@docusaurus/plugin-google-analytics": "3.0.1",
+    "@docusaurus/preset-classic": "3.0.1",
+    "@docusaurus/theme-mermaid": "3.0.1",
     "@loadable/component": "5.15.3",
-    "@mdx-js/react": "1.6.22",
-    "@stackql/docusaurus-plugin-hubspot": "1.1.0",
```

Comment:
Uninstalled this one since it hasn't been updated in forever and wants React 17.